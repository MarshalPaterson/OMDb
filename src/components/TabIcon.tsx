import React, { Component } from 'react';
import {
  Image,
  StyleSheet,
} from 'react-native';

export default class TabIcon extends Component {
  render() {
    const { icon } = this.props;

    return (
      <Image
        source={ icon }
        style={ [ styles.icon ] }
      />
    );
  }
}

const styles = StyleSheet.create({
  icon: {
    width: 26,
    height: 26
  },
});
