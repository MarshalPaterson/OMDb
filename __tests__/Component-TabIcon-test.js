import React from 'react';
import renderer from 'react-test-renderer';
import TabIcon from '../src/components/tabIcon';

jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<TabIcon />).toJSON();
  expect(tree).toMatchSnapshot();
});
