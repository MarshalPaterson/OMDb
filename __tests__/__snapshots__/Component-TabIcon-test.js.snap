// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  style={
    Array [
      Object {
        "height": 26,
        "width": 26,
      },
    ]
  }
/>
`;
